app/api/events.py

```diff
@@ -420,7 +420,6 @@ def before_get_object(self, view_kwargs):
             event = self.session.query(Event).filter_by(identifier=view_kwargs['identifier']).one()
             view_kwargs['id'] = event.id
 
-    decorators = (jwt_required,)
```

Comment:
Why should relationship have no decorators? Is this accessible by everyone? Won't that enable anyone to create a relationship between event and other apis?